### DISCUSSION TITLE
勉強会/OSSコミュニティ向けのグループ機能について

### DISCUSSION THREAD

**Discussion by murakami** [increments/qiita-discussions#18] [要望・アイデア・不具合]
エンジニアフェスタで近日中にコミュニティ向けの機能が用意されると伺いましたが、このディスカッションのクローズドベータの期間は、今月一杯までですので、コミュニティ向けのグループ機能についてあったら嬉しい機能をあげてみます。

- コミュティ単位で記事がまとめられる
- 勉強会などのイベント単位で記事がまとめられる
- メンバ管理などはなくても大丈夫？
  - 誰でも関連づけて記事を投稿できるが善意を期待する？ 
- スケジュール管理が外部のconnpassなどを利用するので不要？
- 告知はconnpassやFacebookグループなどを利用するので不要？

**Comment by kskwtnk** [comment]
@murakami 

ご意見ありがとうございます！

> エンジニアフェスタで近日中にコミュニティ向けの機能が用意されると伺いました

こちらは、Organization機能の話として受け取らせていただきます！
https://qiita.com/organizations

## コミュティ単位で記事がまとめられる

既存のOrganization機能で、おそらくこの領域はカバーできていると思われます！
ただ、好きなOrganizationにいつでも入れる……といった仕組みではないため、もしかしてその点がイメージされている機能と違うのでしょうか？

## 勉強会などのイベント単位で記事がまとめられる

こちらは今はまだ実装されていませんが、話には出ています・
今回のベータテストや本運用のDiscussionsにてvoteが多ければ実装の可能性も高まるかもしれません！

## メンバ管理などはなくても大丈夫？

こちらは既存のOrganization機能にて実装されております！
管理者が招待や削除などを行えます。

## スケジュール管理が外部のconnpassなどを利用するので不要？

イベントを管理するような機能は、初期のうちでは実装予定ではありません……。

## 告知はconnpassやFacebookグループなどを利用するので不要？

告知についても、Qiitaを使って告知ができる！といった機能は話には挙がっておりません……。

**Reply by murakami** [reply]
ありがとうございます。
Organization機能、試してみます。

**Reply by kskwtnk** [reply]
@murakami 

こちらこそありがとうございます！
もし使用いただいた上で疑問ですとか要望ですとか、出てきたら是非Discussionを立てていただきたいです！ 🙏
